### DISCUSSION TITLE
Adding categories to categories

### DISCUSSION THREAD

**Discussion by HuntezWOTB** [stoatchat/discussions#539] [Feature Suggestions]
You all know very well that there are categories - and they include chat rooms and voice channels. I propose to make it so that you can put a category in a category, thereby you can make like a folder in a Windows folder.
It might look something like this:
- Main category
-- Information (Server; Rules; Announcements)
-- Chats (General; General-NSFW; Memes; Sceenshots)
-- Voice chats (Main-1; Main-2; Music)
-- Private (Admins; Moders; Private-Main)
-- Logs (Join; Leave; Kicks; Banned)

Such a thing will make it easier to implement the division of the audience by contigent (Europe, CIS, America, etc.) or other needs.

**Comment by nulldg** [comment]
it's rare that a server may become so large that nested categories are necessary. in such cases, the server may be too large.

**Comment by MasicoreLord** [comment]
Assuming you go through w/ that, there should still be a depth limit, like 2 or 3 levels deep, otherwise it could become difficult to manage, and harder on bots and clients to deal w/.

**Reply by HuntezWOTB** [reply]
I completely agree with you, subcategories should have a maximum depth of 2 folders
